framework/PageInputs/PageMultiSelect.cy.tsx

```diff
@@ -92,32 +92,6 @@ describe('PageMultiSelect', () => {
     cy.get('#test').should('have.text', placeholderText);
   });
 
-  it('should support string options', () => {
```

Comment:
Those values were a problem and were hiding a bug underneath.
You could set the value={organization} but then options = [1,2] which does not overlap.
Because of the casting in the code, it would ignore that the types did not match.